### ISSUE TITLE
build failure on debian

### ISSUE THREAD

**Issue by khimaros** [conundrumer/audioscope#18]
this is an incredible project!

i'm having some difficulty with the initial build on debian bullseye and debian buster with the rust stable toolchain.

i've manually installed the latest stable of libportaudio system wide as suggested:

```
Package: libportaudio2
Version: 19.6.0-1.1

Package: libportaudiocpp0
Version: 19.6.0-1.1
```

I am able to build rust-portaudio from Git (which appears to be 0.7.0):

```
$ git clone https://github.com/RustAudio/rust-portaudio
$ cd rust-portaudio
$ cargo build --release
[...]
    Finished release [optimized] target(s) in 26.70s
```

however, while building (linking) audioscope, i receive the following error:

```
          /usr/bin/ld: /home/user/src/audioscope/target/debug/deps/libportaudio-779a6df6781f5d0f.rlib(pa_jack.o): in function `PaJack_Initialize':
          /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/portaudio-0.7.0/portaudio/src/hostapi/jack/pa_jack.c:800: undefined reference to `jack_client_close'
          /usr/bin/ld: /home/user/src/audioscope/target/debug/deps/libportaudio-779a6df6781f5d0f.rlib(pa_jack.o): in function `BuildDeviceList':
          /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/portaudio-0.7.0/portaudio/src/hostapi/jack/pa_jack.c:617: undefined reference to `jack_port_by_name'
          /usr/bin/ld: /home/user/cargo/registry/src/github.com-1ecc6299db9ec823/portaudio-0.7.0/portaudio/src/hostapi/jack/pa_jack.c:619: undefined reference to `jack_port_get_latency'
          /usr/bin/ld: /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/portaudio-0.7.0/portaudio/src/hostapi/jack/pa_jack.c:596: undefined reference to `jack_port_by_name'
          /usr/bin/ld: /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/portaudio-0.7.0/portaudio/src/hostapi/jack/pa_jack.c:598: undefined reference to `jack_port_get_latency'
[...]
```

happy to provide any other information you need. thanks for releasing this!

**Comment by khimaros** [comment]
i'm seeing the same also on debian buster.

rustc 1.48.0 (7eac88abb 2020-11-16)

stable-armv7-unknown-linux-gnueabihf (default)

**Comment by khimaros** [comment]
https://github.com/conundrumer/audioscope/pull/19 fixes the build on debian/raspbian buster and possibly others by pulling in the latest version of backtrace-rs.

the jack linker errors seem to be the same as https://github.com/RustAudio/rust-portaudio/issues/136 which has a workaround in https://github.com/RustAudio/rust-portaudio/issues/153

**Comment by khimaros** [comment]
i believe this can now be closed since #19 was merged
